### ISSUE TITLE
Ray Tune with old version of YOLO

### ISSUE THREAD

**Issue by farhan-hafizh** [ultralytics/ultralytics#18359]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I have several questions:
- I want to use ray tune for YOLOv10, is it possible?
- When i train YOLOv10 the wandb checkpoint not being saved on wandb.ai web, why is it?
- Also when i train YOLOv10 on kaggle the training history not being logged, instead i get this: DDP: debug command /usr/bin/python3 -m torch.distributed.run --nproc_per_node 2 --master_port 57969 /root/.config/Ultralytics/DDP/_temp_ylnzj7h9132120005425088.py

**Comment by UltralyticsAssistant** [comment]
👋 Hello @farhan-hafizh, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) for helpful resources and examples on getting started with YOLO, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage.

If this is a 🐛 Bug Report, we kindly request a [minimum reproducible example (MRE)](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us investigate the problem more effectively.

If you have ❓ Questions about custom training or specific scenarios, please include as much detail as possible (e.g., scripts, dataset examples, error messages, and training/environment logs) to allow us to better understand and address your issues. Additionally, make sure to follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to maximize your training success.

## Upgrade & Compatibility

It’s important to confirm you’re running the latest `ultralytics` version and dependencies to ensure the issue is not related to out-of-date software. Please upgrade your environment to the latest version using:

```bash
pip install -U ultralytics
```

Verify that your setup meets the necessary [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) for [**Python>=3.8**](https://www.python.org/) and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/).

## Possible Environments

To run YOLO successfully, try any of the following verified environments with the appropriate dependencies preinstalled:

- **Notebooks** with free GPU access:  
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VMs ([GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMIs ([AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Images** for seamless deployments ([Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Distributed Data Parallel (DDP)

For debugging Distributed Data Parallel (DDP) mode issues, ensure your environment complies with distributed training requirements (e.g., correct `torch` and `CUDA` versions). Additional logging or confirmation of your setup may be needed to address the `DDP debug command` message you provided.

## Community Support

If you’d like to connect with others in the Ultralytics community:
- Join our [Discord](https://discord.com/invite/ultralytics) for real-time support 🎧
- Visit our [Discourse](https://community.ultralytics.com) for thought-out discussions 🖋️
- Share knowledge or stories at our [Subreddit](https://reddit.com/r/Ultralytics) 🤝

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

All [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are run daily to validate core functionality across supported platforms. If the badge above is green, it means everything is working as expected on our end ✅.

---

This is an automated response to help you get started quickly. An Ultralytics engineer will review your issue and provide further assistance as soon as possible. Thank you for your patience and understanding! 😊

**Comment by Y-T-G** [comment]
1. Yes. It's the same process as YOLO11.
2. You need to enable wandb by running `!yolo settings wanb=True` in Kaggle before importing Ultralytics
3. Clear your cookies for Kaggle and reload.

**Comment by farhan-hafizh** [comment]
> 1. Yes. It's the same process as YOLO11.
can i tune untrained model like from yaml?

**Comment by Y-T-G** [comment]
Yes
